docs/console/project-configuration/manage-environment-variables/manage-environment-variables-with-akv.md

```diff
@@ -0,0 +1,99 @@
+
+Where:
+
+- `<namespace>` is the namespace where your project will be deployed. it depends on the environment of the project, its value can be retrieved from the [environments page](/console/project-configuration/manage-runtime-environments/index.md) in the Console.
+
+The other fields of the form can be filled in as you wish.
+
+This configuration can also be done from the Azure CLI program `az`, [here](https://azure.github.io/azure-workload-identity/docs/topics/federated-identity-credential.html#federated-identity-credential-for-an-azure-ad-application) is how.
+
+## Setup the Console project
+
+Once the Key Vault instance is configured, we are now ready to set up the Console project. The project set up can be split into three steps:
+
+1. Azure Key Vault provider creation
+2. Project configuration
+3. Microservices configuration
+
+### Azure Key Vault provider
+
+In order to connect to the Azure Key Vault instance you have to create a provider. In the provider creation modal, select the `Azure Key Vault` type when prompted, then fill the fields as described in the [providers management page](/development_suite/set-up-infrastructure/configure-provider.mdx#connect-a-provider)
+
+:::warning
+Please notice that each provider is linked to an Azure Key Vault store, we advice to setup at least two providers for each project, one for production data and another for noprod data, each associated with the respective environment of the project.
+:::
+
+### Project configuration
+
+Connect the provider to the project or environment through the CMS. Retrieve the previously generated provider's `_id` (you can find it inside the `Providers` section of the CMS) and use it to modify the `Environments variables` section in the CMS as follows:
+
+```json
+{
+  "type": "azure-key-vault",
+  "providerId": <your provider's _id>
+}
+```
+
+You also have the option to distinguish between providers for specific environments. This enables you to establish various Azure Key Vault providers for different environments. To achieve this, you need to create an `environmentsVariables` object with the same structure as the one demonstrated above within the target environment object found in the `Environments` section of the project's page in the CMS. Any environment that doesn't specify its own `environmentsVariables` configuration will automatically inherit it from the project.
+
+### Use Azure Key Vault secrets inside the Console
+
+Secrets stored within an Azure Key Vault provider cannot be directly utilized as interpolations in the project's configuration. However, you can access these variables by creating a new microservice environment variable with the value type set to `from secret` the secret name designated as `akv-secret` and the secret key specified as the desired secret name. Detailed instructions for this process can be found in the [microservice configuration section](/development_suite/api-console/api-design/services.md#environment-variable-configuration).
```

Comment:
```suggestion
Secrets stored within an Azure Key Vault provider cannot be directly utilized as interpolations in the project's configuration. However, you can access those variables by creating a new microservice environment variable with the value type set to `from secret` linking a secret with name `akv-secret` and the secret key specified as the desired secret name. Detailed instructions for this process can be found in the [microservice configuration section](/development_suite/api-console/api-design/services.md#environment-variable-configuration).
```

Reply:
I agree with everything but the `those`, I think `these` is more appropriate since we are referring to something close